log/log.go

```diff
@@ -0,0 +1,73 @@
+package log
+
+import (
+	"github.com/Sirupsen/logrus"
+	libmachinelog "github.com/docker/machine/libmachine/log"
```

Comment:
Ah, OK.  I got confused by the pathing here.  I'll have to take a closer look at the arch.